ansible/playbooks/raf/cdap/deploy_cdap.yml

```diff
@@ -43,6 +43,21 @@
   roles:
     - {role: raf/cdap/deploy}
 
+- name: Deploy CDAP-UI Configs
```

Comment:
This section should be in individual playbook and then it would be called here based on flag.